cjio/convert.py

```diff
@@ -142,17 +146,24 @@ def to_glb(cm, do_triangulate=True):
 
     vertexlist = np.array(cm.j["vertices"])
 
-    # gltf uses a right-handed coordinate system. 
-    # glTF defines +Y as up, +Z as forward, and -X as right, thus the front of a glTF 
+    # gltf uses a right-handed coordinate system.
+    # glTF defines +Y as up, +Z as forward, and -X as right, thus the front of a glTF
     # asset faces +Z.
     # The root node of the scene contains the y-up to z-up transformation matrix,
-    # which is needed for inherently z-up data. 
+    # which is needed for inherently z-up data.
     # See https://github.com/CesiumGS/3d-tiles/tree/main/specification#y-up-to-z-up
-    nodes.append({
-        "matrix": [1, 0, 0, 0, 0, 0, -1, 0, 0, 1, 0, 0, 0, 0, 0, 1],
-        "children": [],
-        "name": cm.j.get("metadata", "citymodel").get("identifier", "citymodel")
```

Comment:
this was failing if there were no 'metadata' so I changed it. @hugoledoux can you verify that it is ok?

Reply:
I have no idea, never worked wiht glTF. I didn't write that code...

but cityjson/metadata is not mandatory, so yes it could crash...

but deleting seems wrong (@balazsdukai you know more I reckon?) since it caused a missing matrix. Wouldn't it be better to just put something else for `"name"`?

Reply:
The "name" is not required, so it is safe to omit it.